README.md

```diff
@@ -178,14 +178,14 @@ with conflicting dependencies:
 **gradle**:
 ```groovy
 repositories {
-    maven { url = 'https://oss.sonatype.org/content/repositories/snapshots' }
+    //maven { url = 'https://oss.sonatype.org/content/repositories/snapshots' } // For snapshot builds
```

Comment:
I think we should keep this in there without the comment, it does no harm if present and it might confuse people